.github/workflows/docker_production_image.yml

```diff
@@ -0,0 +1,45 @@
+name: One App Production Image Build and Push
+on:
+  repository_dispatch:
+    types: [one-app-docker-prod-build]
+jobs:
+  one-app-docker-prod-build:
+    name: One App Production Image Build and Push
```

Comment:
can we add a check to request to see if the version is already present ? if it's not continue else exit early ?

Reply:
The tag check action prevents any tags that are already published being published again therefore we can prevent it before it gets to this stage.
Also this would involve pulling images and checking whether they exist, which i believe would be long process and and i believe the tag process would cater for this. Let me know what you think?